src/main/java/com/entropyteam/entropay/common/ReactAdminMapper.java

```diff
@@ -55,14 +57,16 @@ public Filter buildFilter(ReactAdminParams params, Class entityClass) {
                     } else if (isEntityField(entityClass, filter.getKey()) || StringUtils.equalsIgnoreCase(filter.getKey(), SEARCH_TERM_KEY)) {
                         // getList filter
                         getByFieldsFilter.put(filter.getKey(), filter.getValue());
+                    } else if (StringUtils.equalsIgnoreCase(filter.getKey(), DATE_TERM_KEY)) {
+                        getByDateFieldsFilter.put(filter.getKey(), filter.getValue().toString());
```

Comment:
Actually, let's do it a bit more generic, we should receive a fromDate and toDate, instead of the year

Reply:
currently react admin does not allow sending two values from a single input, should we create the startDate and endDate from that value and then add them to the filter?